### DISCUSSION TITLE
== Error ==  system call for: "['...']" finished abnormally, OS return value: 2

### DISCUSSION THREAD

**Discussion by TX0814** [ablab/spades#1009] [Q&A]
Hi! 
I'm running metaspades but it failed.
== Running: /home/cloudam/.conda/envs/spades/bin/spades-core /home/cloudam/raw_data/tx/DAMO_Bacteria_raw/genome/K21/configs/config.info /home/cloudam/raw_data/tx/DAMO_Bacteria_raw/genome/K21/configs/mda_mode.info /home/cloudam/raw_data/tx/DAMO_Bacteria_raw/genome/K21/configs/meta_mode.info

  0:00:00.000     1M / 23M   INFO    General                 (main.cpp                  :  99)   Loaded config from /home/cloudam/raw_data/tx/DAMO_Bacteria_raw/genome/K21/configs/config.info
  0:00:00.007     1M / 23M   INFO    General                 (main.cpp                  :  99)   Loaded config from /home/cloudam/raw_data/tx/DAMO_Bacteria_raw/genome/K21/configs/mda_mode.info
  0:00:00.008     1M / 23M   INFO    General                 (main.cpp                  :  99)   Loaded config from /home/cloudam/raw_data/tx/DAMO_Bacteria_raw/genome/K21/configs/meta_mode.info
  0:00:00.009     1M / 23M   INFO    General                 (memory_limit.cpp          :  54)   Memory limit set to 128 Gb
  0:00:00.009     1M / 23M   INFO    General                 (main.cpp                  : 107)   Starting SPAdes, built from N/A, git revision N/A
  0:00:00.009     1M / 23M   INFO    General                 (main.cpp                  : 108)   Maximum k-mer length: 128
  0:00:00.009     1M / 23M   INFO    General                 (main.cpp                  : 109)   Assembling dataset (/home/cloudam/raw_data/tx/DAMO_Bacteria_raw/genome/dataset.info) with K=21
  0:00:00.009     1M / 23M   INFO    General                 (main.cpp                  : 110)   Maximum # of threads to use (adjusted due to OMP capabilities): 16
  0:00:00.009     1M / 23M   INFO    General                 (pipeline.cpp              : 212)   SPAdes started
  0:00:00.009     1M / 23M   ERROR   General                 (pipeline.cpp              : 216)   Sorry, current version of metaSPAdes can work either with single library (paired-end only) or in hybrid paired-end + (TSLR or PacBio or Nanopore) mode.

== Error ==  system call for: "['/home/cloudam/.conda/envs/spades/bin/spades-core', '/home/cloudam/raw_data/tx/DAMO_Bacteria_raw/genome/K21/configs/config.info', '/home/cloudam/raw_data/tx/DAMO_Bacteria_raw/genome/K21/configs/mda_mode.info', '/home/cloudam/raw_data/tx/DAMO_Bacteria_raw/genome/K21/configs/meta_mode.info']" finished abnormally, OS return value: 2
None

Here are my spades.log and params.txt files from this run:
[spades.log](https://github.com/ablab/spades/files/9209162/spades.log)
[params.txt](https://github.com/ablab/spades/files/9209165/params.txt)

**Comment by asl** [comment]
The log clearly reads:

```
Sorry, current version of metaSPAdes can work either with single library (paired-end only) or in hybrid paired-end + (TSLR or PacBio or Nanopore) mode.
```

**Reply by TX0814** [reply]
So, does it mean that I should merge the forward and reverse of paired-end only into a single library and then use metaSPAdes?

**Reply by asl** [reply]
If your libraries are "compatible" (e.g have similar fragment length distribution) then this might work
